wss/1.0/modules/ROOT/pages/validate-wss-examples.adoc

```diff
@@ -0,0 +1,462 @@
+= Validate Operation Examples
+
+This `validate` operation validates an incoming SOAP request that contains WSS information. The WSS module validates the request against a `WssInboundConfig` configuration.
```

Comment:
The peer `apply` operation intro. uses "The" instead of "This."

Is the following text meant to define what a validate operation is, and then refer specifically to the examples following?